<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositsFrame</class>
 <widget class="QFrame" name="DepositsFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fff; color: #262626;</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>300</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QSpinBox" name="m_timeSpin">
      <property name="geometry">
       <rect>
        <x>116</x>
        <y>110</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>12</number>
      </property>
     </widget>
     <widget class="QLabel" name="m_feeLabel">
      <property name="geometry">
       <rect>
        <x>109</x>
        <y>150</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>110</y>
        <width>57</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Time:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="m_amountSpin">
      <property name="geometry">
       <rect>
        <x>113</x>
        <y>70</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="maximum">
       <double>20000000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>70</y>
        <width>57</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Amount:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_depositButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>196</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Oswald</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #262626; color: #e5e5e5; border-radius: 5px; padding: 6px;</string>
      </property>
      <property name="text">
       <string>DEPOSIT</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_nativeTimeLabel">
      <property name="geometry">
       <rect>
        <x>223</x>
        <y>110</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="m_interestLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="m_feeTextLabel">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>150</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Deposit fee:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_withdrawButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>196</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Oswald</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #262626; color: #e5e5e5; border-radius: 5px; padding: 6px;</string>
      </property>
      <property name="text">
       <string>WIDTHDRAW</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_tickerLabel1">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>70</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true">CCX</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_unlockedDepositLabel">
      <property name="geometry">
       <rect>
        <x>596</x>
        <y>110</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">0.00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>70</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Locked:</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_totalDepositLabel">
      <property name="geometry">
       <rect>
        <x>586</x>
        <y>150</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">0.00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>110</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Unlocked:</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_lockedDepositLabel">
      <property name="geometry">
       <rect>
        <x>596</x>
        <y>70</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">0.00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="m_tickerLabel3">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>150</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true">CCX</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_tickerLabel2">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>110</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true">CCX</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>150</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>20</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Oswald</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CURRENT DEPOSITS</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Oswald</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>NEW DEPOSIT</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="m_depositView">
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>320</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QHeaderView::section {
    background-color: #eee;
    color: #262626;
    font: 12px;
    border: 0px solid black;
   height: 30px;
 text-align: center;

}

QTreeView::item
{
    font: 11px;
    height: 30px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_depositButton</sender>
   <signal>clicked()</signal>
   <receiver>DepositsFrame</receiver>
   <slot>depositClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_withdrawButton</sender>
   <signal>clicked()</signal>
   <receiver>DepositsFrame</receiver>
   <slot>withdrawClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_depositView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>DepositsFrame</receiver>
   <slot>showDepositDetails(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_timeSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DepositsFrame</receiver>
   <slot>timeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_amountSpin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DepositsFrame</receiver>
   <slot>depositParamsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_timeSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DepositsFrame</receiver>
   <slot>depositParamsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>depositClicked()</slot>
  <slot>withdrawClicked()</slot>
  <slot>showDepositDetails(QModelIndex)</slot>
  <slot>timeChanged(int)</slot>
  <slot>depositParamsChanged()</slot>
 </slots>
</ui>
